### PR TITLE
add new formular: rargs

### PR DESCRIPTION
**Pull request by lotabout**
Add `rargs` which tries to combine xargs wtih awk's field reference syntax.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by SMillerDev** [comment]
I think the description should say "like xargs" right?

**Comment by lotabout** [comment]
@SMillerDev Yeah, fixed. Thanks for the review.

File: Formula/rargs.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 "ac6cf3a31ff5b55f86487fa3d3266edf8f562cc6b548d6e636daf373534388ad"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix, "--path", "."
+  end
+
+  test do
+    system "#{bin}/rargs", "-h"
```
Comment (SMillerDev) [inline Formula/rargs.rb]: Is there a test that could be done to test the program works as expected?

Comment (lotabout) [reply Formula/rargs.rb]: A simple test is added.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @lotabout for the pull request

**Pull request closed without merging**